lib/activerecord_adbc_adapter/quoting/sqlite3.rb

```diff
@@ -0,0 +1,16 @@
+require 'active_record/connection_adapters/sqlite3/quoting'
+
+module ActiveRecordADBCAdapter
+  module Quoting
+    class Sqlite3
+      include ActiveRecord::ConnectionAdapters::Quoting
+      include ActiveRecord::ConnectionAdapters::SQLite3::Quoting
```

Comment:
The test passes even with only the `abstract module`, but in that case, the resulting string will have the year and month information removed. (Reference: https://github.com/rails/rails/blob/90a1eaa1b30ba1f2d524e197460e549c03cf5698/activerecord/lib/active_record/connection_adapters/abstract/quoting.rb#L191-L194)


However, SQLite3 explicitly appends `2000-01-01`, so the SQLite3 module is included to maintain this behavior.
(Reference: https://github.com/rails/rails/blob/90a1eaa1b30ba1f2d524e197460e549c03cf5698/activerecord/lib/active_record/connection_adapters/sqlite3/quoting.rb#L74-L77)